order/interfaces.go

```diff
@@ -792,14 +823,77 @@ func (b *Bid) ReservedValue(feeSchedule terms.FeeSchedule,
 	clearingPrice := FixedRatePremium(b.FixedRate)
 
 	return reservedValue(b, func(amt btcutil.Amount) btcutil.Amount {
+		premiumAmt := amt
+		if b.Details().AuctionType == BTCOutboundLiquidity {
+			premiumAmt += b.SelfChanBalance
+		}
+
 		delta, _, _ := takerDelta(
-			feeSchedule, clearingPrice, amt, b.SelfChanBalance,
-			b.LeaseDuration,
+			feeSchedule, clearingPrice, premiumAmt,
+			b.SelfChanBalance, b.LeaseDuration,
 		)
 		return delta
 	}, accountVersion)
 }
 
+// CheckOfferParams makes sure the offer parameters of a sidecar ticket are
+// valid and sane.
+func CheckOfferParams(auctionType AuctionType, capacity, pushAmt,
```

Comment:
The doc for this function refers to sidecar offers specifically. But this PR moves this function from the sidecar package to the order package. Is the doc still correct?